projects/bp-gallery/src/components/views/search/SearchBreadcrumbs.tsx

```diff
@@ -21,7 +22,7 @@ const SearchBreadcrumbs = ({ searchParams }: { searchParams: URLSearchParams })
   const searchParamValues: SearchParams = [];
   const iconForType = (searchType: string) => {
     switch (searchType) {
-      case SearchType.DECADE:
+      case SearchType.TIME_RANGE:
```

Comment:
```suggestion
      case SearchType.DECADE:
      case SearchType.TIME_RANGE:
```

As @BenediktV mentioned on SearchBreadcrumbs.scss file, we should keep the decade-handling and just use the same icon for the time-range search type.